packages/create-platformatic/src/db/create-db-cli.mjs

```diff
@@ -158,8 +158,17 @@ const createPlatformaticDB = async (_args) => {
         spinner.succeed('...done!')
       }
     }
+    await execaNode('./node_modules/@platformatic/db/db.mjs', ['schema', 'config'], { cwd: projectDir })
+    logger.info('Configuration schema successfully created.')
   }
   await askCreateGHAction(logger, env, 'db')
+
+  if (!runPackageManagerInstall) {
+    logger.warn(`You must run the following commands in the project folder to complete the setup:
+    - ${pkgManager} install
+    - npx platformatic db schema config > ./platformatic.config.schema.json
```

Comment:
Is this command correct? I think the file is written automatically by the command (at least that's what's implied by how it's run above with `execa`). If not, the filename in this command should be `platformatic.db.schema.json`.

There's a missing step here too that's implied but not quite clear [in the docs](https://oss.platformatic.dev/docs/next/reference/cli#schema):

- Add `"$schema": "./platformatic.db.schema.json"` to your `platformatic.db.json` config file

There are definitely some things that could be improved/automated around how this works.